static/backbone/js/api/lib/maps/overlays/infobubbles/base.js

```diff
@@ -96,27 +98,43 @@ define(['jquery',
             },
 
             renderViewContent: function () {
-                var template = this.getTemplate("InfoBubbleTemplate");
+                var template = this.getTemplate("InfoBubbleTemplate"),
+                    that = this;
                 this.$el = $(template(this.getContext()));
-
+                this.$el.click(function (e) {
+                    that.bringToFront(e);
+                });
                 this.showUpdatedContent();
             },
-
+            bringToFront: function (e) {
+                var zIndex;
+                //console.log(this.bubble.bubble_.style.zIndex);
+                zIndex = parseInt(this.bubble.bubble_.style.zIndex, 10);
+                this.bubble.bubble_.style.zIndex = zIndex + 1;
+                e.preventDefault();
```

Comment:
This was introduced for an experiment so that kids could look at multiple bubbles at the same time: if one bubble was occluded by another, the kid could just click the occluded bubble and it would appear on top. If it's OK w/you, let's keep this for now, as we might end up supporting multiple bubbles open in parallel. Still an open question.